packages/medusa/src/commands/start.ts

```diff
@@ -109,6 +114,15 @@ async function start(args: {
       const server = GracefulShutdownServer.create(
         http_.listen(port).on("listening", () => {
           logger.success(serverActivity, `Server is ready on port: ${port}`)
+
+          const {
+            admin: { path: adminPath },
```

Comment:
thought: shouldn't we check if the admin is disabled in the config?

Reply:
Yes I think so to, so we don't print a link that goes to no where.

Also think we can skip printing this in production, as the actual link won't be http://localhost:9000/app in that case, and people aren't going to need a link to the admin there anyways.

Reply:
yes thats true 👍

Reply:
I have performed the changes, let me know if that's ok for you

Reply:
Looks good @adrien2p!